shell/core/plugin.ts

```diff
@@ -162,6 +163,12 @@ export class Plugin implements IPlugin {
    * Adds a tab to the UI
    */
   addTab(where: string, when: LocationConfig | string, tab: Tab): void {
+    // tackling https://github.com/rancher/dashboard/issues/11122, we don't want the tab to added in _EDIT view, unless overriden
+    // on extensions side we won't document the mode param for this extension point
```

Comment:
What's the reasoning behind not documenting this? It seems a useful feature that I believe other extension developers would be interested in correct?

Reply:
We don't want them to possibly tamper with an `EDIT` view of a resource like `provisioning.cattle.io.cluster`, imho.

Reply:
Makes sense, we don't need to let them know about the Pandora's box that is `IPlugin` 🙂